app/assets/stylesheets/providers/legal_aid_applications.scss

```diff
@@ -11,6 +11,12 @@ div.proceeding-item {
   border-bottom: 1px solid #cccccc;
 }
 
+div.organisation-item {
```

Comment:
Not a CSS expert (paging @naseberry) but could this not be aliased to the above block?

something like 
```css
div.proceeding-item div.organisation-item {
  margin: 20px 0 0;
  padding: 0;
  border-bottom: 1px solid #cccccc;
}
```

Reply:
You are correct, just missing a comma:

```css
div.organisation-item,
div.proceeding-item {
  margin: 20px 0 0;
  padding: 0;
  border-bottom: 1px solid #cccccc;
}
```